services/predictors/vila/server/api_model.py

```diff
@@ -0,0 +1,42 @@
+"""
+Pydantic equivalents of classes from mmda.types
```

Comment:
These serialize into JSON the same as the `mmda` types, so are only used in the outmost layer of the API request handler.

Gave up trying to inject classes into the `mmda` library that would be usable by this API.